Splash page

sarahlshowers opened the pull request.

sarahlshowers committed:
inital markup and images

sarahlshowers committed:
markup tweaks, alt tags, broken image link, footer

derrickshowers commented on css/stylesheet.css:
```diff
@@ -0,0 +1,18 @@
+/*
```
:+1: for the comments and organization. Nice! :smiley:

derrickshowers commented on index.html:
```diff
@@ -1,14 +1,78 @@
+	<meta charset="UTF-8">
+	<title>Kevin Smith Transportation Group</title>
+	<meta name="description" content="Philadelphia region's newest luxury limoursine and total transportation service.">
+	<meta name="keywords" content="luxury, limo, transportation, Philadelphia">
```
This is perfect for SEO! May want to ask him what keywords he wants here (unless he already provided?)

derrickshowers commented on index.html:
```diff
@@ -1,14 +1,78 @@
+		</div>
+	</header>
+	<section>
+		<h1>Your Luxury Limousine and Total Transportation Service.<br />Arriving July 2015.</h1>
```
I would probably get rid of the `<br />` and make 'Arriving July 2015' an `<h2>` or something else (which should automatically give you the line break)

sarahlshowers replied:
I wondered about that, however it's styled the same in the comp. I'm open to suggestions on how to best format this because I'm not sure how else to do this, but I would think styling H2 the same as H1 isn't a good practice which is why I went with the line break.

derrickshowers replied:
I think if you look at it from a semantic POV, though, the text isn't as important as 'Your Luxury Limousine and Total Transportation Service.' Even if it is, though, it would be better with just duplicating the tag. However, since we don't want two `<h1>`s on the page, an `<h2>` seems like it makes the most sense. Basically, styling shouldn't influence semantics, if that makes sense...

As far as styling, I would just styling with a class (add the same class name to both and override the default style for h1/h2).

sarahlshowers committed:
changing <h1>, remove <br />, add <h2>

sarahlshowers committed:
Merge branch 'master' into splash-page

Conflicts:
	css/stylesheet.css

sarahlshowers committed:
fix merge

sarahlshowers committed:
missing closing <h1>

sarahlshowers committed:
moving images to correct directory, inital page styling

sarahlshowers committed:
added fluid grid, sass, added grid styling to banner and working on aside for info-pannel

sarahlshowers committed:
styling to aside and footer, tweaks to overal pagination

sarahlshowers committed:
complete inital styling for indo-section and footer, added additional styling to center content on larger viewports

derrickshowers commented on landingPages/initialLaunch/css/stylesheet.scss:
```diff
@@ -0,0 +1,162 @@
+    .contact-container {
+      p {
+        padding: 0;
+        // line-height: 10px;
```
remove?

sarahlshowers replied:
Comments removed

derrickshowers commented on landingPages/initialLaunch/index.html:
```diff
@@ -1,14 +1,93 @@
+	</aside>
+
+	<footer>
+		<p>@ 2015 Kevin Smith Transportation Group</p>
```
should this be "@ 2015"? Or a ©?

sarahlshowers replied:
symbol added

derrickshowers commented on landingPages/initialLaunch/index.html:
```diff
@@ -1,14 +1,93 @@
+				<div>
+					<h3>Follow Us</h3>
+					<div class="social-media">
+						<a href="https://www.facebook.com/kevinsmithgroup"><img src="images/icon-facebook.svg" alt="Facebook" height="29" width="29"></a>
```
may want to consider having these open in a new window since they're taking the user away from the page

sarahlshowers replied:
added target="_blank"

derrickshowers commented on landingPages/initialLaunch/index.html:
```diff
@@ -1,14 +1,93 @@
+		<div class="banner-container">
+			<div class="banner row">
+				<figure class="col-1-4">
+					<img src="images/corporate.png" alt="Corporate" height="230" width="280">
```
Not crazy about the way these look when resizing the browser, even slightly. I don't think you need to ensure it looks good at every possible pixel size, but if someone is looking at it on even a slightly smaller screen size, the margins disappear.

One thing to consider is giving the images a class and adding `max-width: 100%` to them. That will keep their widths to 100% of the parent container thus keeping the margin. Try it out and resize the browser--think you might like the way it looks better. :smiley:

sarahlshowers replied:
done and done, implemented max-width-content class for banner, info and headline to render better on larger viewports

derrickshowers commented:
I think you still need to do something about the content extending to far when the page is super wide. The info area above the footer is sorta fixed, but it's still not centered. Looks like you added `margin-top:0 auto`. Should be `margin: 0 auto`.

What I would suggest is adding something like this to your scss file.

``` css
.content {
  margin: 0 auto;
  max-width: 1080px; /* or whatever the max-width of the "content" area is */
}
```

And then add a 'content' class to anything you want to keep inside the box (i.e. all content areas--info footer area, headline text, and whatever else, but I think that's it). Make sense?

sarahlshowers committed:
further styling from peer review

derrickshowers commented on landingPages/initialLaunch/index.html:
```diff
@@ -1,14 +1,93 @@
+	<meta name="keywords" content="luxury, limo, transportation, Philadelphia">
+	<link rel="shortcut icon" href="favicon.ico" type="image/vnd.microsoft.icon">
+	<link href='http://fonts.googleapis.com/css?family=Open+Sans:400italic,400,700' rel='stylesheet' type='text/css'>
+	<link rel="stylesheet" type="text/css" href="css/reset.css">
```
Just noticed this... You can remove the reference to the reset.css since your scss is importing it (so it should all be in stylesheet.css)

sarahlshowers replied:
Removed reference to reset.css from html

derrickshowers commented on landingPages/initialLaunch/index.html:
```diff
@@ -1,14 +1,93 @@
+			</div>
+			<div class="contact-container col-1-2">
+				<div>
+					<h2>Connect with us</h2>
```
indentation seems to be a bit messy

sarahlshowers replied:
Addressed indentation

derrickshowers commented:
Footer is still messed up.

![kevin_smith_transportation_group](https://cloud.githubusercontent.com/assets/1372292/7213784/0f8cd320-e540-11e4-8bfc-be2c634e2cd0.png)

Looks like the grid is overriding that `margin: 0 auto` :disappointed: I guess you'll just have to override that for this instance.

sarahlshowers committed:
removed reset.css from html
addressed indentation
added margin for info-container, grid is overriding styles for largeviewports
